controllers/applicantController.js

```diff
@@ -41,7 +41,34 @@ export async function createExpenseValidationHandler(req, res) {
     }
   }
 
+export const getCompletedRequests = async (req, res) => {
+    const id = parseInt(req.params.id, 10);
+    if (!Number.isInteger(id)) {
+        return res.status(400).json({ error: "Invalid user ID" });
+    }
+    try {
+        const completedRequests = await Applicant.getCompletedRequests(id);
+        if (!completedRequests) {
```

Comment:
The conditional does not work for empty arrays which is the case given in the test, this should change
```suggestion
        if (!completedRequests ||  completedRequests.length === 0) {
```